cmd/describe/cluster/cmd.go

```diff
@@ -133,6 +134,20 @@ func run(_ *cobra.Command, argv []string) {
 		reporter.Errorf("Failed to parse creator ARN for cluster '%s'", clusterKey)
 		os.Exit(1)
 	}
+	phase := ""
+
+	if cluster.State() == cmv1.ClusterStatePending {
+		phase = "(Preparing account)"
+	}
+
+	if cluster.State() == cmv1.ClusterStateInstalling {
+		if !cluster.Status().DNSReady() {
+			phase = "(DNS Setup In progress)"
```

Comment:
I think we should change capitalization to "DNS setup in progress" to keep it consistent

Reply:
updated